queries/rst/highlights.scm

```diff
@@ -20,7 +20,14 @@
   name: (type) @include)
  (#match? @include "^include::$"))
 
-; Blocks
+((directive
+   name: (type) @function.builtin)
+ (#match?
+  @function.builtin
+  ; https://docutils.sourceforge.io/docs/ref/rst/directives.html
+  "^(attention|caution|danger|error|hint|important|note|tip|warning|admonition)|(image|figure)|(topic|sidebar|line-block|parsed-literal|code|math|rubric|epigraph|highlights|pull-quote|compound|container)|(table|csv-table|list-table)|(contents|sectnum|section-numbering|header|footer)|(target-notes)|(meta)|(replace|unicode|date)|(raw|class|role|default-role|title|restructuredtext-test-directive)::$"))
```

Comment:
Not sure if there is a way to split this loooooooong line

Reply:
for me it's fine. There are similar monsters for other languages, also in the upstream queries.